src/fontra/views/editor/panel-selection-info.js

```diff
@@ -484,7 +491,7 @@ export default class SelectionInfoPanel extends Panel {
       for (const [layerName, layerGlyph] of Object.entries(editLayerGlyphs)) {
         layerGlyph.components[componentIndex].transformation = getDecomposedIdentity();
       }
-      return "reset transformation";
+      return translate("sidebar.selection-info.component.transformation.tooltip");
```

Comment:
I think the key should have the word "reset" in it. And I'm not sure it needs the "tooltip" part.

Reply:
done.